### ISSUE TITLE
ui: revamp / re-design event log

### ISSUE THREAD

**Issue by dianasaur323** [cockroachdb/cockroach#17717]
Currently, the event log is often truncated and the events that are surfaced are not comprehensive of all events that might be relevant to cluster performance.

This is a placeholder for work to revamp the event log to make it more valuable to users. The goal is to make it easier for users to correlate changes in cluster behavior to past events.

**Comment by cuongdo** [comment]
I think this would make a good first project for an engineer.

**Comment by cuongdo** [comment]
(once we better define this)

**Comment by dianasaur323** [comment]
ok great! I was hoping to have @josueeee run a design mock when we get to it, but probably still too far out for now.

**Comment by nstewart** [comment]
In the meantime could we just add a tooltip that shows the full text?

**Comment by dianasaur323** [comment]
@nstewart perhaps, but you could also click into the full event log at the bottom to get an expanded view?

**Comment by nstewart** [comment]
If all the events are truncated beyond recognition that component has no value (assuming we added it to tell people what was happening in an actionable way). Asking people to click to an *entirely new page* just to answer a simple question is not good enough UX. 

For example, if I'm on the home page and interested in the item 4 rows down...
![image](https://user-images.githubusercontent.com/894257/29946641-28709298-8e74-11e7-9ec5-b83001bc5507.png)

I have to do the following:
1) try to hover and get frustrated that nothing happens
2) realize there is an option to see all events (on a laptop this may be below the fold which makes this even worse)
3) click "all events"
4) get dropped in a new page with a differently formatted table, losing my original context
![image](https://user-images.githubusercontent.com/894257/29946696-47afb850-8e74-11e7-821e-83a6f6850d98.png)

5) manually count x rows down in this NEW TABLE just to figure out the info I wanted seconds ago.

Moreover, our label components have an inconsistent contract with the user. In certain parts of the dashboard ellipses = hover over to get the rest of the info, like in the node names on the charts

![image](https://user-images.githubusercontent.com/894257/29946800-bc6ded74-8e74-11e7-8dc4-9e5a73d4a3a2.png)

But we have ellipses in the activity stream where hover does nothing.

So long story short clicking to get the full event log isn't good enough. Not saying we change anything in priority, just want to make sure we don't think the current workaround is acceptable

**Comment by dianasaur323** [comment]
@nstewart definitely - I think the event log truncation is a big UX issue. I was worried that we don't have capacity for adding a tooltip before 1.1, so instead of doing that fix, we could do a bigger overhaul in 1.2. I think the events on graphs effort will likely change the importance of this module. Are you okay with fixing these UX issues in 1.2, and sticking with the bad experience of having to click into the full event log?

**Comment by nstewart** [comment]
Cool - I'll defer to you on how and when to solve. You have the best understanding of the team's backlog and capacity.

**Comment by jordanlewis** [comment]
Closing due to age, and we don't have this truncation issue anymore.
